src/lib/connectAxesToLayout.js

```diff
@@ -2,14 +2,19 @@ import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import nestedProperty from 'plotly.js/src/lib/nested_property';
 import {deepCopyPublic, setMultiValuedContainer} from './multiValues';
-import {getDisplayName, localize, capitalize} from '../lib';
+import {capitalize, getAllAxes, getDisplayName, localize} from '../lib';
```

Comment:
Also not new in this PR, but Isn't this `from '.'`? Actually, wouldn't it be better to import from the individual files anyway rather than `lib/index`, since this way is a circular import? It obviously doesn't break, but makes me uneasy...